### ISSUE TITLE
Feature request: linking to source for external modules (specifically Home-Manager)

### ISSUE THREAD

**Issue by llakala** [mynixos/support#18]
MyNixOS provides a link to the source for packages and options under nixpkgs. However, the same functionality doesn't apply for home-manager, where no source is provided. This would be really useful, and is possible, as it's done by https://home-manager-options.extranix.com/.

**Comment by pveierland** [comment]
Agreed! I'll have a look at this when building the updated indexer for MyNixOS 2.0.

**Comment by repparw** [comment]
Bump for this, declarations link would be really useful

**Comment by pveierland** [comment]
Phase 2 of MyNixOS v2 is being built now (package indexing + search + navigation). I'll follow up on this in phase 3 (options indexing + search + navigation) 👍
